src/main/target/STM32F405/target.h

```diff
@@ -32,14 +32,18 @@
 #define USE_I2C_DEVICE_2
 #define USE_I2C_DEVICE_3
 
+#define USE_VCP
+
+#define USE_SOFTSERIAL
+
 #define USE_UART1
 #define USE_UART2
 #define USE_UART3
 #define USE_UART4
 #define USE_UART5
 #define USE_UART6
 
-#define SERIAL_PORT_COUNT       (UNIFIED_SERIAL_PORT_COUNT + 6)
+#define SERIAL_PORT_COUNT       9
```

Comment:
Confirming this number is correct?  Should it include VCP etc?

Reply:
I talked about it with @haslinghuis and I fixed it now.

Reply:
As UNIFIED_SERIAL_PORT_COUNT only was defined after being referenced VCP, SOFTSERIAL and LPUART were not included before.